server/src/main/java/org/elasticsearch/rest/RestResponse.java

```diff
@@ -37,6 +37,7 @@
 public final class RestResponse implements Releasable {
 
     public static final String TEXT_CONTENT_TYPE = "text/plain; charset=UTF-8";
+    public static final Set<String> RESPONSE_PARAMS = Set.of("error_trace");
```

Comment:
It felt right for `RestResponse` to specify which parameters it consumed? not sure if this is the best way to go about it.

Reply:
I think so - this is kinda short-term anyway whilst we look at consolidating the validation steps